schemas/ws/ws_genome_features.yaml

```diff
@@ -0,0 +1,34 @@
+name: ws_genome_features
+type: vertex
+schema:
+  "$schema": "http://json-schema.org/draft-07/schema#"
+  type: object
+  required:
+    - _key
+    - feature_id
+    - workspace_id
+    - object_id
+    - version
+  properties:
+    _key:
+      type: string
+      description: The UPA and feature ID for this data
+      examples: ["35414:73:1_RSP_4039"]
+      pattern: "^\\d+:\\d+:\\d+_\\w*$"  # may need to expand the feature ID part
```

Comment:
To be super safe, we might do a namespace for features just in case we have another "sub-vertex" of the genome with conflicting IDs. Eg. `"\\d+:\\d+:\\d+_ft\\w*$"

Reply:
Per our discussion, we'll leave as is since the collection name acts as a namespace